en/developer_guide/multiple_device.rst

```diff
@@ -0,0 +1,89 @@
+Multiple device
+===============
+
+Prerequisite
+------------
+
+To enable the multi-device rendering, we assume that the server in front of Php (Varnish, Apache2, Nginx, ...)
+ implements the Wurfl librairy and is able to add a request header with the device type.
+
+The device name will be used in the configuration and in the template name following the rule :
+
+* main template : template.html.twig
+* phone template : template.phone.html.twig
```

Comment:
The naming is unclear here. Is "phone" the name of a device ? Is it parametrable ? Even after further reading I'm not sure i get it.